src/backend/ml_api/routers/question.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from fastapi import APIRouter, HTTPException
+from pydantic import BaseModel
+from external_models.transformers_model import models
+from typing import Dict
+
+router = APIRouter()
+
+class QuestionInput(BaseModel):
+    text: str
+    model: str
+    
+class QuestionResponse(BaseModel):
+    response: str
+
+@router.post("/")
+async def question(data: QuestionInput) -> QuestionResponse:
+    try:
+        model = models[data.model]
+        response = model.generate_response(data.text)
```

Comment:
this assumes that the question is already encoded in the `data.text` variable. Is that always the case?